alfresco-dynamic-extensions-repo/alfresco-dynamic-extensions-repo-50/dummy.txt

```diff
@@ -0,0 +1 @@
+To make sure this directory is in git
```

Comment:
Shouldn't this be an empty file named .gitkeep ?

Reply:
These files are removed in a later commit

Reply:
Yeah, noticed that already, more of a global remark